lib/modulr/resources/transactions/transaction.rb

```diff
@@ -4,6 +4,8 @@ module Modulr
   module Resources
     module Transactions
       class Transaction < Base
+        attr_reader :balance, :available_balance
```

Comment:
I updated transaction resoruce to be more consistent with naming and as we allow to read attributes